pilot/pkg/networking/util/util.go

```diff
@@ -257,15 +257,21 @@ func SortVirtualHosts(hosts []route.VirtualHost) {
 	})
 }
 
-// IsProxyVersionGE11 checks whether the given Proxy version is greater than or equals 1.1.
-func IsProxyVersionGE11(node *model.Proxy) bool {
-	ver, _ := node.GetProxyVersion()
-	return ver >= "1.1"
+// IsIstioVersionGE12 checks whether the given Istio version is greater than or equals 1.2.
+func IsIstioVersionGE12(node *model.Proxy) bool {
+	ver, _ := node.GetIstioVersion()
+	return ver >= "1.2"
+}
+
+// IsIstioVersionGE13 checks whether the given Istio version is greater than or equals 1.3.
+func IsIstioVersionGE13(node *model.Proxy) bool {
+	ver, _ := node.GetIstioVersion()
+	return ver >= "1.3"
 }
 
 // IsXDSMarshalingToAnyEnabled controls whether "marshaling to Any" feature is enabled.
 func IsXDSMarshalingToAnyEnabled(node *model.Proxy) bool {
-	return IsProxyVersionGE11(node) && !features.DisableXDSMarshalingToAny
+	return IsIstioVersionGE13(node) && !features.DisableXDSMarshalingToAny
```

Comment:
Shouldn't we just remove the version check? now any marshalling cannot be used with Istio 1.2 which is not the case

Reply:
Do we officially saying do not support versions before 1.1?

Reply:
We only support one release for backward compatibility. We do not support all the way back to 1.1

Reply:
yeah this will break Any marshalling for 1.2 though. We don't support 1.1 so we just need to remove the check, we don't need to move the check to >1.3

Reply:
That is to say if we release 1.3, the 1.1 is not maintained, and users upgraded control plane to 1.3, they have to upgrade the proxies meanwhile, right?

Reply:
@rshriram this still needs to be changed. it should be `return !features.DisableXDSMarshalingToAny` - we don't need the version check